csm/core/blogic/const.py

```diff
@@ -562,7 +562,7 @@
 DECRYPTION_KEYS = {
     "CHANNEL>password": "sspl",
     "S3>ldap_password": "openldap",
-    "CSM>password": "system"
+    "CSM>password": "cortx"
```

Comment:
we should not hard code instead we sould read it from template file
https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/127893714/Cipher+key+generation+for+secret+password

Reply:
Okay. I will make this change in seperate PR